pkg/http/api.go

```diff
@@ -642,14 +642,14 @@ func (a *api) onCreateActorReminder(reqCtx *fasthttp.RequestCtx) {
 		respondWithError(reqCtx, fasthttp.StatusInternalServerError, msg)
 		log.Debug(msg)
 	} else {
-		respondEmpty(reqCtx, fasthttp.StatusOK)
+		respondEmpty(reqCtx, fasthttp.StatusCreated)
```

Comment:
Generally the expectation with a 201 is that you've created some restful resource and will return its URL in the location header or it's representation in the response body.  https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/201

201 is useful for cases where the caller won't know the URL of the thing they're creating ahead of time.

If we're not returning any info maybe we should be.

Reply:
Hmm, then we aren't consistent with the other 201 status cases we have today. We never return a body on a 201. So we either need we need to only ever return 200, or always give a URL in the body for all cases. I don't have a strong opinion on which was, I was just trying to ensure we are at least consistent with our own API.

Reply:
I think typically the URL to the newly created resource should be returned in the Location header. The body can contain the representation of the newly created resource or it can be empty.

Reply:
Right, my reading of the code suggests we're doing neither of those things.

Reply:
Confirmed, we are neither giving a location header, nor a body on a 201 anywhere in the API. Given that none of our clients appear to actually need this, should I just make all 201s 200s and call it a day?

Reply:
After an offline chat, we determined that the intended behavior is not to have a location header or body on these replies. I'm changing this to a 204.